### ISSUE TITLE
Launch emulator with arguments

### ISSUE THREAD

**Issue by Lahiri** [TBemme/galaxy-integration-psp#1]
Hello, 

I am using Retroarch as emulator and I need to launch it with arguments to select the emulation core.

I tried to configure config.py like this, but it does not seem to work:

emu_path = r'"E:/Emulation/RetroArch/retroarch.exe" -L "cores/ppsspp_libretro.dll"'

Is this not supported yet?

**Comment by TBemme** [comment]
Hi, thanks for the issue report, I did not consider this.

I've changed a little how the launching function works, so this should be supported now. There is an additional option in the config.py for arguments if required. You will need to download the updated plugin.py and config.py files (or add a new emu_args line to your existing config.py). So, if you have:

```
emu_path = r"E:/Emulation/RetroArch/retroarch.exe"
emu_args = [ r"-L", r"E:/Emulation/RetroArch/cores/ppsspp_libretro.dll" ]
```
I think that should work.
